common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -361,8 +363,10 @@ const PageLayoutComponent: FunctionComponent<Props> = ({
 
         {/* The no javascript version of the burger menu relies on the footer being present on the page,
         as we then use an anchor link to take people to the navigation links in the footer.
-        We only completely remove the footer if you've got JS. */}
-        {(!hideFooter || !isEnhanced) && <Footer venues={venues} />}
+        We only completely remove the footer if you've got JS. If we've hidden the header, then we don't need to worry about this because the navigation links aren't there at all */}
+        {(!hideFooter || (!isEnhanced && !hideHeader)) && (
+          <Footer venues={venues} />
+        )}
```

Comment:
Double checking this as the only way to change cookie preferences is in the footer and it might be legal requirement? (We hide it in the Item Viewer only at the moment)

Reply:
I'm going to have another look at this – I'm suddenly not very sure of the logic I was going for anyhow. Thanks. Is it ok that we hide the footer in the item viewer then, or are you saying that maybe it isn't?

Reply:
Hiding it on the Item Viewer still was a conscious call!
This is the chat we had about it https://github.com/wellcomecollection/wellcomecollection.org/issues/10714, worth checking with Lauren, maybe we'll want to re-add the idea of a minimal footer?

Reply:
I don't think we want a minimal footer for these pages – I'd say they're supposed to feel almost like a modal with the previous/next buttons flush to the bottom of the screen, so any kind of footer would ruin that experience

Reply:
Those pages _are_ similar to item viewer in that they feel more like an overlay than a proper page, so it probably makes sense to hide it there as well. (fyi @LaurenFBaily )

Reply:
going to revert it to how it was for now